src/pages-and-resources/data/supportedPages.js

```diff
@@ -0,0 +1,2 @@
+const supportedPages = new Set(['calculator', 'discussion']);
+export default supportedPages;
```

Comment:
I feel this construction is very clunky, suggestions are very welcome.

Reply:
I don't think this is needed at all. The page will either have a settings link, or a legacy link, or no link at all. If it has a legacy link, show that, if it doesn't show the settings link.

Reply:
Done!